docker/docker-compose.yaml

```diff
@@ -40,3 +40,16 @@ services:
       - ..:/code:delegated
       - ~/.m2:/root/.m2:delegated
     entrypoint: /bin/bash
+
+  cross-compile-aarch64-runtime-setup:
+    image: netty:cross_compile_aarch64
+    build:
+      context: .
+      dockerfile: Dockerfile.cross_compile_aarch64
+      args:
+        gcc_version : "4.9-2016.02"
+
+  cross-compile-aarch64:
+    image: netty:cross_compile_aarch64
+    depends_on: [cross-compile-aarch64-runtime-setup]
+    command: /bin/bash -cl "pushd netty/transport-native-unix-common && ../mvnw clean install -Plinux-aarch64 && popd && pushd netty/transport-native-epoll && ../mvnw clean install -Plinux-aarch64 -DskipTests && popd"
```

Comment:
@wangxiyuan why are we skipping tests here?

Reply:
Since this is cross compile, the test environment is still on X86. So when run test, it still require `libnetty_tcnative_linux_x86_64.so` which will lead the error like

```
java.lang.IllegalArgumentException: Failed to load any of the given libraries: [netty_tcnative_linux_x86_64_fedora, netty_tcnative_linux_x86_64, netty_tcnative_x86_64, netty_tcnative]
```

Reply:
can you add a comment explaining that? something along the lines of..

# since we are cross compiling netty_tcnative as arch64 it cannot be loaded on x64 -> skipTests

Reply:
I added a suggested change.